plugin.xml

```diff
@@ -70,7 +70,8 @@ SOFTWARE.
             </intent-filter>
         </config-file>
         <source-file src="src/android/io/branch/BranchSDK.java" target-dir="src/io/branch" />
-        <framework src="io.branch.sdk.android:library:1.+" />
+        <framework src="io.branch.sdk.android:library:1.+" custom="true" />
```

Comment:
is `custom="true"` required here? this is what's breaking my build of the test bed. i removed it - and it rebuilds just fine.